package.json

```diff
@@ -43,6 +43,16 @@
 					"group": "openInNewWindowGroup"
 				}
 			]
+		},
+		"configuration": {
+			"title": "Open in New Window configuration",
+			"properties": {
+				"open-in-new-window.showNotification": {
+					"type": "boolean",
+					"description": "Show notification",
+					"default": false
```

Comment:
> It seems unnecessary

https://github.com/onatm/open-in-new-window/issues/1#issue-735415012

Therefore `false` as a default value

Reply:
It is a sensible default value.